src/bin/syncerd.rs

```diff
@@ -32,29 +32,20 @@ extern crate log;
 use clap::Clap;
 
 use farcaster_node::syncerd::{self, Opts};
-use farcaster_node::Config;
-
-fn main() {
-    println!("syncerd: blockchain tasks management microservice");
+use farcaster_node::{Error, ServiceConfig};
 
+fn main() -> Result<(), Error> {
```

Comment:
What's up with the return type?

Reply:
What is the problem about returning an error?

Reply:
This function doesn't return, right?

Reply:
The binary returns eventually, so exit code is not zero and error is printed. You can have main return result in rust :)

Reply:
When I mean eventually: it returns early if it miss some params to start

Reply:
> The binary returns eventually, so exit code is not zero and error is printed. You can have main return result in rust :)

But which statement within this main function actually returns? I only see a panic case here, and no `?`, `return` or omitted `;`. So I'm guessing you are just returning for style? But even then, it seems inconsistent with `swapd.rs` and `peerd.rs`.

Reply:
Oh sorry, I guess the code that was there got removed, run fn can panic but it is handled by the expect so yes I have to removed that. (On the phone, didn’t open the file before now to check)

Reply:
Done in 9e3a5fef5cc0610d9b79137e86706d6a1b7ff378